index.html

```diff
@@ -23,10 +37,9 @@
   </head>
 
   <body>
-    <div id="google_translate_element"></div>
     <video autoplay="" loop="" mediagroup="5" playsinline="" muted="" poster="" id="bgvid" class="hide-on-med-and-down"><source src="img/embarcaWebm.webm" type="video/webm"></video>
     <div id="countdownBanner">
-      <p>🚀 🚀 Puertas de embarque abiertas, quedan  <span id="cddays"></span> Días <span id="cdhours"></span> Horas <span id="cdminutes"></span> Minutos <span id="cdseconds"></span> Segundos  🚀 🚀 <a href="#" class="modal-trigger" data-target="modal2">Inscribirme</a>
+      <p>🚀 🚀 Puertas de embarque abiertas, quedan <span id="cddays"></span> Días <span id="cdhours"></span> Horas <span id="cdminutes"></span> Minutos <span id="cdseconds"></span> Segundos 🚀 🚀 <a href="https://www.f6s.com/embarca2018-02/apply" target="_blank">Inscribirme</a>
```

Comment:
Fran, aca tenemos un conflicto. 
```suggestion
      <p>🚀 🚀 Puertas de embarque abiertas, quedan <span id="cddays"></span> Días <span id="cdhours"></span> Horas <span id="cdminutes"></span> Minutos <span id="cdseconds"></span> Segundos 🚀 🚀 <a href="https://www.f6s.com/embarca2018-02/apply" target="_blank">Inscribirme</a>
```

Fran fijate que esto lo agregue yo porque me lo paso el gonza. Habria que cargar un remote dentro de tu repo, y hacer el pull. Nos juntamos te enseño y lo hacemos.